scripts/init-poktrolld.sh

```diff
@@ -1,35 +1,80 @@
-#!/bin/sh
+#!/bin/bash
 
+set -e
+
+# Install required packages
+apt-get update
+apt-get install -y wget jq
+
+# Can change the branch from `pocket-network-genesis` repo to test before merging to master
```

Comment:
Reminder to use TODO_IN_THIS_PR for self comments like this in the future.

Please remove before merging.

Reply:
Oh, this is not a comment line for ME, but for anybody who wants to test the branch from `pocket-network-genesis` repo.

I thought I'd avoid a `#PUC` with that comment here, haha.

So I would like to keep it. The default branch is `master` of `pocket-network-genesis` repo.

Reply:
I'll merge it with this comment, but if you feel strongly - let me know and I'll remove it.